addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/tests.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+  <TITLE>
+    Automation Framework - Job Tests
+  </TITLE>
+</HEAD>
+<BODY>
+<H1>Automation Framework - Job Outcome Tests</H1>
+The automation framework supports testing the job outcomes based on conditions that you can set in the YAML file.
+Currently, the only type of tests available are statistical ones. More types will be added in the future.
+
+<H2>Statistics</H2>
+Statistics-based tests are universally supported by all add-ons that have an automation job. If there is a relevant
+statistic used by an add-on, a test can be created with it.
```

Comment:
Note for the team - should we document all the stats somewhere eg on the website??